pipelines/merge/Snakefile

```diff
@@ -0,0 +1,97 @@
+import pandas as pd
+import os
+from pathlib import Path
+import pyfaidx
+
+fa = pyfaidx.Fasta(config["fasta_path"])
+contigs = list(fa.keys())
+
+fofn = pd.read_csv(config["fofn"], sep="\t")
+samples = fofn["sample"].unique()
+
+
+rule all_merge:
+  input:
+    expand(
+      os.path.join("merged.zarr/{chrom}/calldata/{field}"),
+      field=["GQ", "AD", "GT"], 
```

Comment:
It is a little complicated. It won't work with the `combine_zarr` script as is unfortunately. I think it's worth working through though. 
Perhaps:
```
fields=["calldata/GT", "variants/MQ"] 
```
Might be a good option

Reply:
I think the `combine_zarr` script could be modified to take `--input-field` and `--output-field` arguments, instead of just `--field`. So in this case you could run `combined_zarr.py --input-field=variants/MQ --output-field=calldata/MQ`. If that makes sense, feel free to make that change in this PR.

Reply:
I think I'll do this in a separate PR, I think it's involved enough to split out. Brandy is using this branch to run her QC analyses.